api/api_sources/resources/csv/MechanicalTreatmentIssueCode.csv

```diff
@@ -1,11 +1,11 @@
-mechanical_treatment_issue_code,description
-NA,None
-FC,Fencing
-AW,Adverse weather
-PL,Private land
-RB,River bank
-ST,Stability 
-SR,Silt release
-RA,Range animals
-WO,Wash out
-WF,Wildfire 
+treatment_issues_code,description

```

Comment:
Try to keep two-five characters code for each entry to make it consistent with existing code tables.